src/main.cpp

```diff
@@ -5242,8 +5242,8 @@ bool static ProcessMessage(CNode* pfrom, string strCommand, CDataStream& vRecv,
     {
         pfrom->SetRecvVersion(min(pfrom->nVersion, PROTOCOL_VERSION));
 
-        // Mark this node as currently connected, so we update its timestamp later.
-        if (pfrom->fNetworkNode) {
+        {
+            // Mark this node as currently connected, so we update its timestamp later.
```

Comment:
I believe we should be addrman.Connected(pnode->addr) on inbound connections too. I could document this better in the commit message.